Tasks/Gradle/Gradle.ps1

```diff
@@ -89,7 +89,8 @@ if ($jdkPath)
     Write-Verbose "JAVA_HOME set to $env:JAVA_HOME"
 }
 
-$buildRootPath = Split-Path $wrapperScript -Parent
+$buildRootPath = $cwd
+$wrapperDirectory = Split-Path $wrapperScript -Parent
```

Comment:
You can give build for sub-modules only. However the gradlew exist on main module directory